evm/src/cpu/cpu_stark.rs

```diff
@@ -114,7 +206,7 @@ pub fn ctl_arithmetic_base_rows<F: Field>() -> TableWithColumns<F> {
     // the third input.
     TableWithColumns::new(
         Table::Cpu,
-        ctl_data_ternops(&OPS, false),
```

Comment:
We want to add `COL_MAP.op.syscall` + `COL_MAP.op.exception` + `COL_MAP.op.prover_input` to the OPS list. This can be done with the `Column::sum` API, but will require a reworking of `ctl_data_ternops`.

This way we don't have to have a separate cross-table lookup just for range checks.